integration/dockerfiles/Dockerfile_test_metadata

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/distroless/base@sha256:628939ac8bf3f49571d05c6c76b8688cb4a851af6c7088e599388259875bde20
+FROM debian:10.2
```

Comment:
ok to change this one from distroless/base to debian?

Reply:
since there is no reason to use distroless for the test case. Distroless is only available on GCR and there is an issue with GCR when you're not authenticated with gcloud.